### DISCUSSION TITLE
light curve fitting

### DISCUSSION THREAD

**Discussion by kathmcp** [phoebe-project/phoebe2#495] [General]
Hi again,

I have managed to get run_compute to work on my model, and it has produced the following figure. The blue curve actually looks like a decent shape for my observations, but the fluxes, and flux variations are obviously too high. Have I missed something major, that it is so far out? Or do I need to just keep adjusting the parameters until it fits better. I can see changes happening as I adjust requiv (primary), mass ratio and sma, but they are fairly minor. 
![Plot](https://user-images.githubusercontent.com/81473857/125813537-91518106-404f-4811-9682-07ea4e8cf96d.PNG)

**Comment by kecnry** [comment]
The main thing that affects the general flux levels is the [passband luminosities](http://phoebe-project.org/docs/2.3/tutorials/pblum) (although depending on which mode you choose, temperatures and radii also play a significant role). Once you get those to the same scale, it should be easier to compare models against your observations.

**Reply by helenkurd** [reply]
hello sir in phoebe 0.32 version how can i fit light curve plot to find initial parameters suh as (mass ratio, surface potential, temperature, Gravity Darkening, Limb Darkening ,Reflection, Inclination)

**Reply by kecnry** [reply]
see https://github.com/phoebe-project/phoebe1/discussions/16

**Reply by helenkurd** [reply]
hello sir in phoebe 2.3 how can i start to plot light curve and i have data observation of phase and flux in excell file?

**Reply by gecheline** [reply]
for adding observations see [Datasets Tutorial](http://phoebe-project.org/docs/2.3/tutorials/datasets), for plotting see [Plotting Tutorial](http://phoebe-project.org/docs/2.3/tutorials/plotting )
